scripts/ie11-compat-check.sh

```
@@ -0,0 +1,96 @@
+# Warn-only per team decision: this script always exits 0 so the Bacon suite
+# doesn't block merges while we build trust in the signal. Bacon status is
+# also OPTIONAL as belt-and-suspenders.
+
+# Early diagnostic so we see the script started even if setup.sh explodes.
+echo "[ie11-compat-check] script starting; OKTA_HOME=${OKTA_HOME:-<unset>} REPO=${REPO:-<unset>}"
+
+# Bacon-side setup (present in CI, absent locally). setup.sh sets `set -eo
+# pipefail` which we relax right after so the `cmd; STATUS=$?` capture pattern
+# below works and L2 runs even if L1.5 flags issues.
+if [ -n "${OKTA_HOME:-}" ] && [ -f "${OKTA_HOME}/${REPO:-}/scripts/setup.sh" ]; then
+  source "${OKTA_HOME}/${REPO}/scripts/setup.sh"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)